config/config.toml.example

```diff
@@ -135,12 +135,12 @@ cluster-ssl-key = ""
 # If enable status report HTTP service.
 report-status = true
 
-# TiDB status host.
-status-host = "0.0.0.0"
-
 ## status-host is the HTTP address for reporting the internal status of a TiDB server, for example:
 ## API for prometheus: http://${status-host}:${status_port}/metrics
 ## API for pprof:      http://${status-host}:${status_port}/debug/pprof
+# TiDB status host.
```

Comment:
Actually `TiDB status host` only have one more information than `status-host`, which is `tidb`... I think we can remove this useless comment. So as the `tidb status port` comment.

We should explain more about the `status-host` and `status-port` in the above comment block.